tests/Fixtures/Scratch/NullRef31.yaml

```diff
@@ -22,8 +24,10 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/repository'
                 description: 'The repository'
+                oneOf:
+                  - { $ref: '#/components/schemas/repository', description: 'The repository' }
```

Comment:
Not sure this duplication of the description is correct. Originally it is next to `$ref` so it does't really apply to the `null`. OTOH, we'll never have any other `oneOf` values...

If we just keep the on next to `$ref` then line 370 can be simplified to just `$ref = ['oneOf' => [$ref]];` ...